tests/experimental/pydantic/test_conversion.py

```diff
@@ -844,52 +839,6 @@ class UserType:
     assert user.passwords == ["hunter2"]
 
 
-def test_sort_creation_fields():
```

Comment:
I dropped the `DataclassCreationField` it is no longer needed with the new `StrawberryObejct` field creation hook.

Reply:
@patrick91 do you approve this? also when #1187 is merged this is probably useless.